src/wallet/rpcwallet.cpp

```diff
@@ -4555,6 +4555,30 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
             throw JSONRPCError(RPC_INVALID_PARAMETER, string("Invalid parameter, duplicated address: ") + addrStr);
         };
 
+        std::optional<libzcash::UnifiedAddress> ua = std::nullopt;
+        if (std::holds_alternative<libzcash::UnifiedAddress>(decoded.value())) {
+            ua = std::get<libzcash::UnifiedAddress>(decoded.value());
```

Comment:
I guess I'm not surprised that this works, but I don't think this is an idiom that we've used before (assignment of a pointer to a std::optional lvalue)